packages/react-ds/src/components/Input/Hook.tsx

```diff
@@ -0,0 +1,12 @@
+import React, { useState } from 'react'
+
+interface IHook {
+  children: Function
+}
+export default ({ children: Children }: IHook) => {
```

Comment:
Since you didn't provide a description for us to get a overview of the pull request, could you explain why is there a function that deals with form state here?